frameworks/agno/research-agent/server.py

```
@@ -0,0 +1,46 @@
+        )
+
+    # Setup logging after environment checks
+    logging.basicConfig(
+        stream=sys.stdout,
+        level=logging.INFO,
+        format="[%(levelname)-5.5s] %(message)s",
+    )
+
+    autostart()
+
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)